packages/authenticator/amplify_authenticator/lib/src/l10n/generated/message_localizations.dart

```diff
@@ -108,6 +110,18 @@ abstract class AuthenticatorMessageLocalizations {
   /// In en, this message translates to:
   /// **'A confirmation code has been sent.'**
   String get codeSentUnknown;
+
+  /// The message that is displayed after a value was copied to the clipboard
+  ///
+  /// In en, this message translates to:
+  /// **'Copied to clipboard!'**
+  String get copySucceeded;
```

Comment:
I feel like `totpCopyKey` (`copyKey`) should live right next to these

Reply:
This is the correct location. Localization is grouped by widget type, not function.